src/commands/dump.rs

```diff
@@ -150,7 +150,10 @@ async fn get_databases(cli: &mut Connection)
     -> Result<Vec<String>, anyhow::Error>
 {
     let mut query = cli.query(
-        "SELECT sys::Database.name",
+        "WITH
```

Comment:
You can write this as short as:
You could do shorter:

```suggestion
        "SELECT (sys::Database FILTER NOT .builtin).name"
```

Reply:
Sure, fixed